cargo/private/cargo_build_script.bzl

```diff
@@ -3,6 +3,7 @@
 load("@bazel_skylib//lib:paths.bzl", "paths")
 load("@bazel_tools//tools/build_defs/cc:action_names.bzl", "CPP_COMPILE_ACTION_NAME", "C_COMPILE_ACTION_NAME")
 load("@bazel_tools//tools/cpp:toolchain_utils.bzl", "find_cpp_toolchain")
+load("//cargo:features.bzl", "SYMLINK_EXEC_ROOT_FEATURE", "feature_enabled")
```

Comment:
Why would this need to be a part of the public api vs living in `//cargo/private`?

Reply:
That sounds reasonable, but could you then move the `feature_enabled` logic into `//cargo/private`? Surely that doesn't seem necessary, right?

Reply:
Done.